src/blockchain/src/blockchain.rs

```diff
@@ -88,6 +88,12 @@ impl Debug for Blockchain {
 impl Blockchain {
     pub fn new(keypair: &identity::ed25519::Keypair) -> Self {
         let local_id = PeerId::from(identity::PublicKey::Ed25519(keypair.public()));
+        let genesis_pub_key: [u8; 44] = [
```

Comment:
where is the private key of this pub key?

Reply:
Dump where is this being shared?